blackbox/docs/cli-tools.rst

```diff
@@ -1,209 +1,229 @@
 
-      sh$ crate-node unsafe-bootstrap
 
-``detach-cluster``
-^^^^^^^^^^^^^^^^^^
+.. _cli-crate-node-commands:
 
-The ``detach-cluster`` command lets you move nodes from one cluster to
-another. You can also move nodes into a cluster that you have created using
-``unsafe-bootstrap`` command. If ``unsafe-bootstrap`` was not possible, it
-also lets you move nodes into a brand-new cluster.
+Commands
+--------
 
-  .. code-block:: console
++----------------------+------------------------------------------------------+
+| Command              | Description                                          |
++======================+======================================================+
+| ``repurpose``        | Clean up any unnecessary data on disk after changing |
+|                      | the role of a node.                                  |
++----------------------+------------------------------------------------------+
+| ``unsafe-bootstrap`` | Force the election of a master and create a new      |
+|                      | cluster in the event of losing the majority of       |
+|                      | master-eligible nodes.                               |
++----------------------+------------------------------------------------------+
+| ``detach-cluster``   | Detach a node from a cluster so that it can join a   |
+|                      | new one.                                             |
++----------------------+------------------------------------------------------+
+
+
+.. _cli-crate-node-options:
+
+Options
+-------
+
++---------------------+-----------------------------------------------------+
+| Option              | Description                                         |
++=====================+=====================================================+
+| ``--ordinal <INT>`` | Specify which node to target if there is more than  |
+|                     | one node sharing a data path                        |
++---------------------+-----------------------------------------------------+
+| ``-E <KV_PAIR>``    | Configures a setting using a key-value (*KV*) pair  |
+|                     |                                                     |
++---------------------+-----------------------------------------------------+
+| ``-h, --help``      | Return all of the command parameters                |
++---------------------+-----------------------------------------------------+
+| ``-s, --silent``    | Show minimal output                                 |
++---------------------+-----------------------------------------------------+
+| ``-v, --verbose``   | Shows verbose output                                |
++---------------------+-----------------------------------------------------+
 
-      sh$ crate-node detach-cluster
+.. SEEALSO::
 
-.. _Troubleshooting with crate-node CLI: https://crate.io/docs/crate/guide/en/latest/best-practices/crate-node.html
+    The ``crate-node`` command is modeled on the `elasticsearch-node`_ command.
```

Comment:
Please don't refer to elastic, we ported over the documentation because we don't want to refer to them.